01week/resume/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8" />
+  <title>Resume</title>
+  <link rel="stylesheet" href="./css/style.css" />
+</head>
+
+<body>
+  <h1>Resume</h1>
+  <p>
+    David Kroll 1100 College Ave Norman, OK 73072 Email: dkroll713@gmail.com
+  </p>
+  <h2>Education</h2>
+  <h3>University of Oklahoma, 2015-2019</h3>
+  <p>
+    Bachelor's of Accounting
+  <br />
+    Bachelor's of Management Information Systems
+  </p>
+  <h3>Austin Coding Academy</h3>
+  <p>
+    Website stuff
+  </p>
+  <h2>Relevant Experience</h2>
+  <p>
+    Court Clerk, Oklahoma City Juvenile Drug Court, 2017-2018
+  </p>
+    <ul>
+      <li>
+      Assist attorneys with gathering and organizing court documents
+      </li>
+      <li>
+        Become familiar with clients, cases
+      </li>
+      <li>
+        Other general office work
+      </li>
+    </ul>
+  <h2>Skills and Accomplishments</h2>
+  <p>
+    Proficient in the entire Microsoft Office Suite, Visual Basic, exposure to C# <br /> Proficient in HTML, CSS, Javascript <br /> Learning more programming languages <br /> Leadership experience; captain of football team in high school and freshman
+    year of college <br /> Recieved a Presidential Service Award w/ Emphasis in Economic Inequality for work with Houston Food Bank
+  </p>
+  <h2>References</h2>
+  <p>
+    my mom
+  </p>
+</body>
+
+</html>
```

Comment:
This file shouldn't be in the bio branch! See if you can figure out how to get this untracked.